roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -115,8 +115,8 @@
     - osd_group_name is defined
     - osd_group_name in group_names
     - raw_multi_journal
-    - raw_journal_devices is not defined
-      or devices is not defined
+    - raw_journal_devices|length == 0
+      or devices|length == 0
```

Comment:
LGTM, can we just rephrase the fail sentence for clarity with something like: `please provide devices and raw journal devices to your osd scenario`?

Thanks!